README.md

```diff
@@ -70,13 +70,13 @@ And here is an example of how to run the ServiceNow sync with [Faros Cloud](http
   --k8s-mem-limit 500Mi \
   --keep-containers
 ```
-**Note**: To execute this command successfully, `kubectl` should be installed in your local environment and the kubernetes cluster context and credentials should be already configured. Namespace, cpu, and mem limits parameters are optional. `--keep-containers` parameter allows to keep the pod on the cluster after the source/destination containers complete the sync.
+**Note**: To execute this command successfully, `kubectl` should be installed in your local environment and the Kubernetes cluster context and credentials should be already configured. Namespace, cpu, and mem limits parameters are optional. `--keep-containers` parameter allows to keep the pod on the cluster after the source/destination containers complete the sync.
```

Comment:
1. lets change to just:
```
**Note**: To execute this command successfully, Kubernetes cluster context and credentials should be already configured.
```

2. let's remove `--k8s-mem-limit` and `--k8s-cpu-limit` and use `--max-cpus` and `--max-mem` instead

Reply:
3. lets document `--keep-containers` in the README

Reply:
> let's remove --k8s-mem-limit and --k8s-cpu-limit and use --max-cpus and --max-mem instead

I wanted to reuse them too - but they are provided in a different format (`M` vs `Mi` for Mbytes) and we also redefine the value immediately when parsing params:  
```
            --max-mem)
                max_memory="-m $2"
                shift 2 ;;
            --max-cpus)
                max_cpus="--cpus $2"
                shift 2 ;;
```
Hence I thought it would be too confusing for user

Reply:
I think we can handle the conversion. IMHO, having less parameters is better